mne/viz/evoked.py

```diff
@@ -277,6 +277,14 @@ def _plot_evoked(evoked, picks=None, exclude='bads', unit=True, show=True,
                            ' for interactive SSP selection.')
 
     _check_option('gfp', gfp, [True, False, 'only'])
+    _check_option('spatial_colors', spatial_colors, [True, False, 'auto'])
+
+    # Check channel location
+    if spatial_colors == 'auto':
+        if _check_ch_locs(info=info):
+            spatial_colors = True
+        else:
+            spatial_colors = False
```

Comment:
simpler:

```suggestion
        spatial_colors = _check_ch_locs(info=info)
```

Reply:
Thx, changed the code according to your suggestions.